   0.0 TEL | Telepresence 0.93 launched at Wed Nov  7 09:21:17 2018
   0.0 TEL |   /usr/bin/telepresence --deployment tel
   0.0 TEL | Platform: linux
   0.0 TEL | Python 3.6.6 (default, Jul 19 2018, 14:25:17)
   0.0 TEL | [GCC 8.1.1 20180712 (Red Hat 8.1.1-5)]
   0.0 TEL | [1] Running: uname -a
   0.0   1 | Linux localhost.localdomain 4.18.16-200.fc28.x86_64 #1 SMP Sat Oct 20 23:53:47 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux
   0.0 TEL | BEGIN SPAN main.py:41(main)
   0.0 TEL | BEGIN SPAN startup.py:58(__init__)
   0.0 TEL | [2] Capturing: kubectl version --short
   0.1 TEL | [3] Capturing: kubectl config current-context
   0.2 TEL | [4] Capturing: kubectl config view -o json
   0.4 TEL | [5] Capturing: kubectl get ns nemo
   0.5 TEL | Command: kubectl 1.12.2
   0.5 TEL | Context: nemo/127-0-0-1:8443/system:admin, namespace: nemo, version: 1.11.0+d4cacc0
   0.5 TEL | END SPAN startup.py:58(__init__)    0.5s
   0.5 TEL | [6] Capturing: ssh -V
   0.6 TEL | [7] Running: sudo -n echo -n
   0.7 >>> | Starting proxy with method 'vpn-tcp', which has the following limitations: All processes are affected, only one telepresence can run per machine, and you can't use other VPNs. You may need to add cloud hosts and headless services with --also-proxy. For a full list of method limitations see https://telepresence.io/reference/methods.html
   0.7 >>> | Volumes are rooted at $TELEPRESENCE_ROOT. See https://telepresence.io/howto/volumes.html for details.
   0.7 TEL | [8] Capturing: kubectl --context nemo/127-0-0-1:8443/system:admin --namespace nemo get pods telepresence-connectivity-check --ignore-not-found
   1.7 TEL | Scout info: {'latest_version': '0.93', 'application': 'telepresence', 'notices': []}
   1.7 TEL | BEGIN SPAN remote.py:149(get_remote_info)
   1.7 TEL | BEGIN SPAN remote.py:73(get_deployment_json)
   1.7 TEL | [9] Capturing: kubectl --context nemo/127-0-0-1:8443/system:admin --namespace nemo get deployment -o json --export tel
   1.8 TEL | END SPAN remote.py:73(get_deployment_json)    0.1s
   1.8 TEL | Searching for Telepresence pod:
   1.8 TEL |   with name tel-*
   1.8 TEL |   with labels {'app': 'tel'}
   1.8 TEL | [10] Capturing: kubectl --context nemo/127-0-0-1:8443/system:admin --namespace nemo get pod -o json --export
   2.0 TEL | Checking identity-v0-6-75c97dd794-bdr8c
   2.0 TEL | --> Name does not match
   2.0 TEL | Checking links-v0-7-7c54ff8844-zr5pd
   2.0 TEL | --> Name does not match
   2.0 TEL | Checking mongo-links-888499877-bmtgd
   2.0 TEL | --> Name does not match
   2.0 TEL | Checking pg-79d8c67d77-wz885
   2.0 TEL | --> Name does not match
   2.0 TEL | Checking tel-75d469c475-l2mbs
   2.0 TEL | Looks like we've found our pod!
   2.0 TEL | BEGIN SPAN remote.py:111(wait_for_pod)
   2.0 TEL | [11] Capturing: kubectl --context nemo/127-0-0-1:8443/system:admin --namespace nemo get pod tel-75d469c475-l2mbs -o json
   2.0 TEL | END SPAN remote.py:111(wait_for_pod)    0.1s
   2.0 TEL | END SPAN remote.py:149(get_remote_info)    0.3s
   2.0 TEL | BEGIN SPAN __init__.py:38(connect)
   2.0 TEL | [12] Launching kubectl logs: kubectl --context nemo/127-0-0-1:8443/system:admin --namespace nemo logs -f tel-75d469c475-l2mbs --container telepresence
   2.0 TEL | [13] Launching kubectl port-forward: kubectl --context nemo/127-0-0-1:8443/system:admin --namespace nemo port-forward tel-75d469c475-l2mbs 44447:8022
   2.0 TEL | [14] Running: ssh -F /dev/null -q -oStrictHostKeyChecking=no -oUserKnownHostsFile=/dev/null -p 44447 telepresence@localhost /bin/true
   2.1 TEL | [14] exit 255 in 0.01 secs.
   2.1  12 | Listening...
   2.1  12 | 2018-11-07T07:15:30+0000 [-] Loading ./forwarder.py...
   2.1  12 | 2018-11-07T07:15:31+0000 [-] /etc/resolv.conf changed, reparsing
   2.1  12 | 2018-11-07T07:15:31+0000 [-] Resolver added ('172.30.0.2', 53) to server list
   2.1  12 | 2018-11-07T07:15:31+0000 [-] SOCKSv5Factory starting on 9050
   2.1  12 | 2018-11-07T07:15:31+0000 [socks.SOCKSv5Factory#info] Starting factory <socks.SOCKSv5Factory object at 0x7f1c6d0eb4e0>
   2.1  12 | 2018-11-07T07:15:31+0000 [-] DNSDatagramProtocol starting on 9053
   2.1  12 | 2018-11-07T07:15:31+0000 [-] Starting protocol <twisted.names.dns.DNSDatagramProtocol object at 0x7f1c6d0eb860>
   2.1  12 | 2018-11-07T07:15:31+0000 [-] Loaded.
   2.1  12 | 2018-11-07T07:15:31+0000 [twisted.scripts._twistd_unix.UnixAppLogger#info] twistd 18.7.0 (/usr/bin/python3.6 3.6.5) starting up.
   2.1  12 | 2018-11-07T07:15:31+0000 [twisted.scripts._twistd_unix.UnixAppLogger#info] reactor class: twisted.internet.epollreactor.EPollReactor.
   2.1  12 | 2018-11-07T07:16:02+0000 [Poll#error] Failed to contact Telepresence client:
   2.1  12 | 2018-11-07T07:16:02+0000 [Poll#error] An error occurred while connecting: 99: Address not available.
   2.1  12 | 2018-11-07T07:16:02+0000 [Poll#warn] Perhaps it's time to exit?
   2.1  12 | 2018-11-07T07:16:32+0000 [Poll#error] Failed to contact Telepresence client:
   2.1  12 | 2018-11-07T07:16:32+0000 [Poll#error] An error occurred while connecting: 99: Address not available.
   2.1  12 | 2018-11-07T07:16:32+0000 [Poll#warn] Perhaps it's time to exit?
   2.1  12 | 2018-11-07T07:17:02+0000 [Poll#error] Failed to contact Telepresence client:
   2.1  12 | 2018-11-07T07:17:02+0000 [Poll#error] An error occurred while connecting: 99: Address not available.
   2.1  12 | 2018-11-07T07:17:02+0000 [Poll#warn] Perhaps it's time to exit?
   2.1  12 | 2018-11-07T07:17:32+0000 [Poll#error] Failed to contact Telepresence client:
   2.1  12 | 2018-11-07T07:17:32+0000 [Poll#error] An error occurred while connecting: 99: Address not available.
   2.1  12 | 2018-11-07T07:17:32+0000 [Poll#warn] Perhaps it's time to exit?
   2.1  12 | 2018-11-07T07:18:02+0000 [Poll#error] Failed to contact Telepresence client:
   2.1  12 | 2018-11-07T07:18:02+0000 [Poll#error] An error occurred while connecting: 99: Address not available.
   2.1  12 | 2018-11-07T07:18:02+0000 [Poll#warn] Perhaps it's time to exit?
   2.1  12 | 2018-11-07T07:18:32+0000 [Poll#error] Failed to contact Telepresence client:
   2.1  12 | 2018-11-07T07:18:32+0000 [Poll#error] An error occurred while connecting: 99: Address not available.
   2.1  12 | 2018-11-07T07:18:32+0000 [Poll#warn] Perhaps it's time to exit?
   2.1  12 | 2018-11-07T07:19:02+0000 [Poll#error] Failed to contact Telepresence client:
   2.1  12 | 2018-11-07T07:19:02+0000 [Poll#error] An error occurred while connecting: 99: Address not available.
   2.1  12 | 2018-11-07T07:19:02+0000 [Poll#warn] Perhaps it's time to exit?
   2.1  12 | 2018-11-07T07:19:32+0000 [Poll#error] Failed to contact Telepresence client:
   2.1  12 | 2018-11-07T07:19:32+0000 [Poll#error] An error occurred while connecting: 99: Address not available.
   2.1  12 | 2018-11-07T07:19:32+0000 [Poll#warn] Perhaps it's time to exit?
   2.1  12 | 2018-11-07T07:20:02+0000 [Poll#error] Failed to contact Telepresence client:
   2.1  12 | 2018-11-07T07:20:02+0000 [Poll#error] An error occurred while connecting: 99: Address not available.
   2.1  12 | 2018-11-07T07:20:02+0000 [Poll#warn] Perhaps it's time to exit?
   2.1  12 | 2018-11-07T07:20:32+0000 [Poll#error] Failed to contact Telepresence client:
   2.1  12 | 2018-11-07T07:20:32+0000 [Poll#error] An error occurred while connecting: 99: Address not available.
   2.1  12 | 2018-11-07T07:20:32+0000 [Poll#warn] Perhaps it's time to exit?
   2.1  12 | 2018-11-07T07:21:02+0000 [Poll#error] Failed to contact Telepresence client:
   2.1  12 | 2018-11-07T07:21:02+0000 [Poll#error] An error occurred while connecting: 99: Address not available.
   2.1  12 | 2018-11-07T07:21:02+0000 [Poll#warn] Perhaps it's time to exit?
   2.1  12 | 2018-11-07T07:21:10+0000 [stdout#info] Set DNS suffix we filter out to: [(b'localdomain',)]
   2.1  12 | 2018-11-07T07:21:10+0000 [stdout#info] Result for b'hellotelepresence9.localdomain' is ['127.0.0.1']
   2.1  12 | 2018-11-07T07:21:10+0000 [stdout#info] Result for b'hellotelepresence10.localdomain' is ['127.0.0.1']
   2.2  13 | Forwarding from 127.0.0.1:44447 -> 8022
   2.2  13 | Forwarding from [::1]:44447 -> 8022
   2.3 TEL | [15] Running: ssh -F /dev/null -q -oStrictHostKeyChecking=no -oUserKnownHostsFile=/dev/null -p 44447 telepresence@localhost /bin/true
   2.3  13 | Handling connection for 44447
   2.5 >>> | 
   2.5 >>> | No traffic is being forwarded from the remote Deployment to your local machine. You can use the --expose option to specify which ports you want to forward.
   2.5 >>> | 
   2.5 TEL | Launching Web server for proxy poll
   2.5 TEL | [16] Launching SSH port forward (socks and proxy poll): ssh -N -oServerAliveInterval=1 -oServerAliveCountMax=10 -F /dev/null -q -oStrictHostKeyChecking=no -oUserKnownHostsFile=/dev/null -p 44447 telepresence@localhost -L127.0.0.1:38775:127.0.0.1:9050 -R9055:127.0.0.1:42627
   2.5 TEL | END SPAN __init__.py:38(connect)    0.5s
   2.5 TEL | BEGIN SPAN remote_env.py:65(get_remote_env)
   2.5 TEL | [17] Capturing: kubectl --context nemo/127-0-0-1:8443/system:admin --namespace nemo exec tel-75d469c475-l2mbs --container telepresence -- python3 -c 'import json, os; print(json.dumps(dict(os.environ)))'
   2.5  13 | Handling connection for 44447
   3.6 TEL | [17] captured in 1.05 secs.
   3.6 TEL | END SPAN remote_env.py:65(get_remote_env)    1.0s
   3.6 TEL | BEGIN SPAN mount.py:32(mount_remote_volumes)
   3.6 TEL | [18] Capturing: sshfs -p 44447 -F /dev/null -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null telepresence@localhost:/ /tmp/tel-k12zhehx/fs
   3.6  13 | Handling connection for 44447
   3.8 TEL | END SPAN mount.py:32(mount_remote_volumes)    0.2s
   3.8 TEL | BEGIN SPAN vpn.py:235(connect_sshuttle)
   3.8 TEL | BEGIN SPAN vpn.py:79(get_proxy_cidrs)
   3.8 TEL | END SPAN vpn.py:79(get_proxy_cidrs)    0.0s
   3.8 TEL | [19] Launching sshuttle: sshuttle-telepresence -v --dns --method nat -e 'ssh -oStrictHostKeyChecking=no -oUserKnownHostsFile=/dev/null -F /dev/null' --to-ns 127.0.0.1:9053 -r telepresence@localhost:44447 172.30.0.0/16
   3.8 TEL | BEGIN SPAN vpn.py:276(connect_sshuttle,sshuttle-wait)
   3.8 TEL | [20] Capturing: python3 -c 'import socket; socket.gethostbyname('"'"'hellotelepresence0'"'"')'
   3.9  20 | Traceback (most recent call last):
   3.9  20 |   File "<string>", line 1, in <module>
   3.9  20 | socket.gaierror: [Errno -2] Name or service not known
   3.9 TEL | [20] exit 1 in 0.09 secs.
   4.0 TEL | [21] Capturing: python3 -c 'import socket; socket.gethostbyname('"'"'hellotelepresence1'"'"')'
   4.1  21 | Traceback (most recent call last):
   4.1  21 |   File "<string>", line 1, in <module>
   4.1  21 | socket.gaierror: [Errno -2] Name or service not known
   4.1 TEL | [21] exit 1 in 0.08 secs.
   4.2 TEL | [22] Capturing: python3 -c 'import socket; socket.gethostbyname('"'"'hellotelepresence2'"'"')'
   4.2  19 | Starting sshuttle proxy.
   4.3  22 | Traceback (most recent call last):
   4.3  22 |   File "<string>", line 1, in <module>
   4.3  22 | socket.gaierror: [Errno -2] Name or service not known
   4.3 TEL | [22] exit 1 in 0.10 secs.
   4.4 TEL | [23] Capturing: python3 -c 'import socket; socket.gethostbyname('"'"'hellotelepresence3'"'"')'
   4.5  23 | Traceback (most recent call last):
   4.5  23 |   File "<string>", line 1, in <module>
   4.5  23 | socket.gaierror: [Errno -2] Name or service not known
   4.5 TEL | [23] exit 1 in 0.10 secs.
   4.6 TEL | [24] Capturing: python3 -c 'import socket; socket.gethostbyname('"'"'hellotelepresence4'"'"')'
   4.7  24 | Traceback (most recent call last):
   4.7  24 |   File "<string>", line 1, in <module>
   4.7  24 | socket.gaierror: [Errno -2] Name or service not known
   4.7 TEL | [24] exit 1 in 0.08 secs.
   4.7  19 | firewall manager: Starting firewall with Python version 3.6.6
   4.7  19 | firewall manager: ready method name nat.
   4.7  19 | IPv6 enabled: False
   4.7  19 | UDP enabled: False
   4.7  19 | DNS enabled: True
   4.7  19 | TCP redirector listening on ('127.0.0.1', 12300).
   4.7  19 | DNS listening on ('127.0.0.1', 12300).
   4.7  19 | Starting client with Python version 3.6.6
   4.7  19 | c : connecting to server...
   4.7  13 | Handling connection for 44447
   4.7  19 | Warning: Permanently added '[localhost]:44447' (ECDSA) to the list of known hosts.
   4.8 TEL | [25] Capturing: python3 -c 'import socket; socket.gethostbyname('"'"'hellotelepresence5'"'"')'
   4.9  25 | Traceback (most recent call last):
   4.9  25 |   File "<string>", line 1, in <module>
   4.9  25 | socket.gaierror: [Errno -2] Name or service not known
   4.9 TEL | [25] exit 1 in 0.09 secs.
   5.0 TEL | [26] Capturing: python3 -c 'import socket; socket.gethostbyname('"'"'hellotelepresence6'"'"')'
   5.1  26 | Traceback (most recent call last):
   5.1  26 |   File "<string>", line 1, in <module>
   5.1  26 | socket.gaierror: [Errno -2] Name or service not known
   5.1 TEL | [26] exit 1 in 0.14 secs.
   5.2 TEL | [27] Capturing: python3 -c 'import socket; socket.gethostbyname('"'"'hellotelepresence7'"'"')'
   5.4  27 | Traceback (most recent call last):
   5.4  27 |   File "<string>", line 1, in <module>
   5.4  27 | socket.gaierror: [Errno -2] Name or service not known
   5.4 TEL | [27] exit 1 in 0.15 secs.
   5.5 TEL | [28] Capturing: python3 -c 'import socket; socket.gethostbyname('"'"'hellotelepresence8'"'"')'
   5.6  28 | Traceback (most recent call last):
   5.6  28 |   File "<string>", line 1, in <module>
   5.6  28 | socket.gaierror: [Errno -2] Name or service not known
   5.6 TEL | [28] exit 1 in 0.11 secs.
   5.7 TEL | [29] Capturing: python3 -c 'import socket; socket.gethostbyname('"'"'hellotelepresence9'"'"')'
   5.8  29 | Traceback (most recent call last):
   5.8  29 |   File "<string>", line 1, in <module>
   5.8  29 | socket.gaierror: [Errno -2] Name or service not known
   5.8 TEL | [29] exit 1 in 0.13 secs.
   5.9 TEL | [30] Capturing: python3 -c 'import socket; socket.gethostbyname('"'"'hellotelepresence10'"'"')'
   5.9  19 | Starting server with Python version 3.6.5
   5.9  19 |  s: latency control setting = True
   5.9  19 |  s: available routes:
   5.9  19 |  s:   2/172.17.0.0/16
   5.9  19 | c : Connected.
   5.9  19 | firewall manager: setting up.
   5.9  19 | >> iptables -t nat -N sshuttle-12300
   5.9  19 | >> iptables -t nat -F sshuttle-12300
   5.9  19 | >> iptables -t nat -I OUTPUT 1 -j sshuttle-12300
   6.0  19 | >> iptables -t nat -I PREROUTING 1 -j sshuttle-12300
   6.0  19 | >> iptables -t nat -A sshuttle-12300 -j RETURN --dest 127.0.0.1/32 -p tcp
   6.0  19 | >> iptables -t nat -A sshuttle-12300 -j REDIRECT --dest 172.30.0.0/16 -p tcp --to-ports 12300 -m ttl ! --ttl 42
   6.0  19 | >> iptables -t nat -A sshuttle-12300 -j REDIRECT --dest 10.100.102.1/32 -p udp --dport 53 --to-ports 12300 -m ttl ! --ttl 42
   6.0  19 | >> iptables -t nat -A sshuttle-12300 -j REDIRECT --dest 224.0.0.252/32 -p udp --dport 5355 --to-ports 12300 -m ttl ! --ttl 42
   6.0  19 | conntrack v1.4.4 (conntrack-tools): 0 flow entries have been deleted.
   6.0  19 | c : DNS request from ('10.100.102.2', 54125) to None: 49 bytes
   6.0  12 | 2018-11-07T07:21:23+0000 [stdout#info] Result for b'hellotelepresence10.localdomain' is ['127.0.0.1']
   6.0 TEL | [31] Capturing: python3 -c 'import socket; socket.gethostbyname('"'"'hellotelepresence11'"'"')'
   6.1  19 | c : DNS request from ('10.100.102.2', 44765) to None: 49 bytes
   6.1  12 | 2018-11-07T07:21:23+0000 [stdout#info] Result for b'hellotelepresence11.localdomain' is ['127.0.0.1']
   6.1 TEL | END SPAN vpn.py:276(connect_sshuttle,sshuttle-wait)    2.3s
   6.1 TEL | END SPAN vpn.py:235(connect_sshuttle)    2.3s
   6.1 TEL | Everything launched. Waiting to exit...
   6.1 TEL | BEGIN SPAN __init__.py:530(wait_for_exit)
   7.4  19 | c : DNS request from ('10.100.102.2', 36622) to None: 34 bytes
   7.4  12 | 2018-11-07T07:21:25+0000 [stdout#info] A query: b'www.google.co.il'
   7.7  19 | c : DNS request from ('10.100.102.2', 42559) to None: 35 bytes
   7.7  12 | 2018-11-07T07:21:25+0000 [stdout#info] A query: b'beacons4.gvt2.com'
   8.4  19 | c : Accept TCP: 10.100.102.2:49846 -> 172.30.39.119:443.
   8.4  19 | c : Accept TCP: 10.100.102.2:49848 -> 172.30.39.119:443.
   8.4  19 | c : Accept TCP: 10.100.102.2:49850 -> 172.30.39.119:443.
   8.4  19 | c : Accept TCP: 10.100.102.2:49864 -> 172.30.39.119:443.
   8.4  19 | c : Accept TCP: 10.100.102.2:49866 -> 172.30.39.119:443.
   8.5  19 | c : Accept TCP: 10.100.102.2:49876 -> 172.30.39.119:443.
   8.5  19 | c : SW#10:10.100.102.2:49848: uwrite: got EPIPE
   8.5  19 |  s: SW'unknown':Mux#6: deleting (11 remain)
   8.5  19 |  s: SW#9:172.30.39.119:443: deleting (10 remain)
   8.5  19 | c : SW#8:10.100.102.2:49846: uwrite: got EPIPE
   8.5  19 | c : Accept TCP: 10.100.102.2:49882 -> 172.30.39.119:443.
   8.5  19 | c : Accept TCP: 10.100.102.2:49884 -> 172.30.39.119:443.
   8.5  19 | c : Accept TCP: 10.100.102.2:49886 -> 172.30.39.119:443.
   8.5  19 | c : SW#10:10.100.102.2:49848: deleting (17 remain)
   8.5  19 | c : SW#10:10.100.102.2:49848: error was: nowrite: [Errno 107] Transport endpoint is not connected
   8.5  19 | c : SW'unknown':Mux#6: deleting (16 remain)
   8.5  19 | c : SW#12:10.100.102.2:49864: deleting (15 remain)
   8.5  19 | c : SW#12:10.100.102.2:49864: error was: uread: [Errno 104] Connection reset by peer
   8.5  19 | c : SW'unknown':Mux#8: deleting (14 remain)
   8.5  19 | c : SW'unknown':Mux#5: deleting (13 remain)
   8.5  19 | c : SW#8:10.100.102.2:49846: deleting (12 remain)
   8.5  19 | c : SW#8:10.100.102.2:49846: error was: nowrite: [Errno 107] Transport endpoint is not connected
   8.6  19 | c : Accept TCP: 10.100.102.2:49912 -> 172.30.39.119:443.
   8.7  19 | c : Accept TCP: 10.100.102.2:49918 -> 172.30.39.119:443.
   8.8  19 |  s: SW'unknown':Mux#5: deleting (19 remain)
   8.8  19 |  s: SW#8:172.30.39.119:443: deleting (18 remain)
   8.8  19 |  s: SW'unknown':Mux#8: deleting (17 remain)
   8.8  19 |  s: SW#11:172.30.39.119:443: deleting (16 remain)
   8.8  19 |  s: SW'unknown':Mux#7: deleting (15 remain)
   8.8  19 |  s: SW#10:172.30.39.119:443: deleting (14 remain)
   8.8  19 |  s: SW'unknown':Mux#9: deleting (13 remain)
   8.8  19 |  s: SW#12:172.30.39.119:443: deleting (12 remain)
   8.8  19 | c : SW#14:10.100.102.2:49876: uwrite: got EPIPE
   8.8  19 | c : SW#14:10.100.102.2:49876: deleting (15 remain)
   8.8  19 | c : SW#14:10.100.102.2:49876: error was: nowrite: [Errno 107] Transport endpoint is not connected
   8.8  19 | c : SW'unknown':Mux#10: deleting (14 remain)
   8.8  19 | c : Accept TCP: 10.100.102.2:49924 -> 172.30.39.119:443.
   8.9  19 | c : warning: closed channel 11 got cmd=TCP_DATA len=31
   8.9  19 | c : warning: closed channel 12 got cmd=TCP_DATA len=31
   8.9  19 | c : warning: closed channel 15 got cmd=TCP_DATA len=31
   9.0  19 |  s: warning: closed channel 15 got cmd=TCP_STOP_SENDING len=0
   9.0  19 | c : warning: closed channel 15 got cmd=TCP_EOF len=0
  12.4  19 | c : DNS request from ('10.100.102.2', 36622) to None: 34 bytes
  12.4  12 | 2018-11-07T07:21:30+0000 [stdout#info] A query: b'www.google.co.il'
  12.4  12 | 2018-11-07T07:21:30+0000 [stdout#info] getaddrinfo error: [Errno -3] Try again
  12.4  19 | c : DNS request from ('10.100.102.2', 58313) to None: 46 bytes
  12.4  12 | 2018-11-07T07:21:30+0000 [stdout#info] Updated query of type 1 from b'www.google.co.il.localdomain' to b'www.google.co.il'
  12.4  12 | 2018-11-07T07:21:30+0000 [stdout#info] A query: b'www.google.co.il'
  12.7  19 | c : DNS request from ('10.100.102.2', 42559) to None: 35 bytes
  12.7  12 | 2018-11-07T07:21:30+0000 [stdout#info] A query: b'beacons4.gvt2.com'
  12.7  19 | c : DNS request from ('10.100.102.2', 36460) to None: 47 bytes
  12.7  12 | 2018-11-07T07:21:30+0000 [stdout#info] getaddrinfo error: [Errno -3] Try again
  12.7  12 | 2018-11-07T07:21:30+0000 [stdout#info] Updated query of type 1 from b'beacons4.gvt2.com.localdomain' to b'beacons4.gvt2.com'
  12.7  12 | 2018-11-07T07:21:30+0000 [stdout#info] A query: b'beacons4.gvt2.com'
  13.4  19 | c : DNS request from ('10.100.102.2', 33301) to None: 34 bytes
  13.4  12 | 2018-11-07T07:21:31+0000 [stdout#info] A query: b'www.google.co.il'
  13.7  19 | c : DNS request from ('10.100.102.2', 33024) to None: 56 bytes
  13.7  12 | 2018-11-07T07:21:31+0000 [stdout#info] A query: b'cuscochromeextension-pa.googleapis.com'
  13.7  19 | c : DNS request from ('10.100.102.2', 40850) to None: 35 bytes
  13.7  12 | 2018-11-07T07:21:31+0000 [stdout#info] A query: b'beacons4.gvt2.com'
  13.7 TEL | (proxy checking local liveness)
  13.7  12 | 2018-11-07T07:21:31+0000 [Poll#info] Checkpoint
  13.8  19 | c : DNS request from ('10.100.102.2', 48398) to None: 37 bytes
  13.8  12 | 2018-11-07T07:21:31+0000 [stdout#info] A query: b'clients4.google.com'
  17.4  12 | 2018-11-07T07:21:35+0000 [stdout#info] getaddrinfo error: [Errno -3] Try again
  17.4  19 | c : DNS request from ('10.100.102.2', 58313) to None: 46 bytes
  17.4  12 | 2018-11-07T07:21:35+0000 [stdout#info] Updated query of type 1 from b'www.google.co.il.localdomain' to b'www.google.co.il'
  17.4  12 | 2018-11-07T07:21:35+0000 [stdout#info] A query: b'www.google.co.il'
  17.4  12 | 2018-11-07T07:21:35+0000 [stdout#info] getaddrinfo error: [Errno -3] Try again
  17.4  12 | 2018-11-07T07:21:35+0000 [stdout#info] Failed to lookup b'www.google.co.il' due to [Failure instance: Traceback (failure with no frames): <class 'twisted.names.error.DomainError'>: [Errno -3] Try again
  17.4  12 | 2018-11-07T07:21:35+0000 [stdout#info] ], falling back to b'www.google.co.il.localdomain'
  17.4  12 | 2018-11-07T07:21:35+0000 [-] DNSDatagramProtocol starting on 5039
  17.4  12 | 2018-11-07T07:21:35+0000 [-] Starting protocol <twisted.names.dns.DNSDatagramProtocol object at 0x7f1c6cf99400>
  17.7  19 | c : DNS request from ('10.100.102.2', 36460) to None: 47 bytes
  17.7  12 | 2018-11-07T07:21:35+0000 [stdout#info] getaddrinfo error: [Errno -3] Try again
  17.7  12 | 2018-11-07T07:21:35+0000 [stdout#info] Updated query of type 1 from b'beacons4.gvt2.com.localdomain' to b'beacons4.gvt2.com'
  17.7  12 | 2018-11-07T07:21:35+0000 [stdout#info] A query: b'beacons4.gvt2.com'
  17.8  12 | 2018-11-07T07:21:35+0000 [stdout#info] getaddrinfo error: [Errno -3] Try again
  17.8  12 | 2018-11-07T07:21:35+0000 [stdout#info] Failed to lookup b'beacons4.gvt2.com' due to [Failure instance: Traceback (failure with no frames): <class 'twisted.names.error.DomainError'>: [Errno -3] Try again
  17.8  12 | 2018-11-07T07:21:35+0000 [stdout#info] ], falling back to b'beacons4.gvt2.com.localdomain'
  17.8  12 | 2018-11-07T07:21:35+0000 [-] DNSDatagramProtocol starting on 29493
  17.8  12 | 2018-11-07T07:21:35+0000 [-] Starting protocol <twisted.names.dns.DNSDatagramProtocol object at 0x7f1c6cf996d8>
  18.4  19 | c : DNS request from ('10.100.102.2', 33301) to None: 34 bytes
  18.4  12 | 2018-11-07T07:21:36+0000 [stdout#info] A query: b'www.google.co.il'
  18.5  12 | 2018-11-07T07:21:36+0000 [stdout#info] getaddrinfo error: [Errno -3] Try again
  18.5  12 | 2018-11-07T07:21:36+0000 [-] DNSDatagramProtocol starting on 31508
  18.5  12 | 2018-11-07T07:21:36+0000 [-] Starting protocol <twisted.names.dns.DNSDatagramProtocol object at 0x7f1c6d04b4e0>
  18.5  12 | 2018-11-07T07:21:36+0000 [-] (UDP Port 5039 Closed)
  18.5  12 | 2018-11-07T07:21:36+0000 [-] Stopping protocol <twisted.names.dns.DNSDatagramProtocol object at 0x7f1c6cf99400>
  18.6  19 | c : DNS request from ('10.100.102.2', 59611) to None: 46 bytes
  18.6  12 | 2018-11-07T07:21:36+0000 [stdout#info] Updated query of type 1 from b'www.google.co.il.localdomain' to b'www.google.co.il'
  18.6  12 | 2018-11-07T07:21:36+0000 [stdout#info] A query: b'www.google.co.il'
  18.7  19 | c : DNS request from ('10.100.102.2', 33024) to None: 56 bytes
  18.7  12 | 2018-11-07T07:21:36+0000 [stdout#info] A query: b'cuscochromeextension-pa.googleapis.com'
  18.7  12 | 2018-11-07T07:21:36+0000 [stdout#info] getaddrinfo error: [Errno -3] Try again
  18.7  19 | c : DNS request from ('10.100.102.2', 35541) to None: 68 bytes
  18.7  12 | 2018-11-07T07:21:36+0000 [stdout#info] Updated query of type 1 from b'cuscochromeextension-pa.googleapis.com.localdomain' to b'cuscochromeextension-pa.googleapis.com'
  18.7  19 | c : DNS request from ('10.100.102.2', 40850) to None: 35 bytes
  18.7  12 | 2018-11-07T07:21:36+0000 [stdout#info] A query: b'cuscochromeextension-pa.googleapis.com'
  18.7  12 | 2018-11-07T07:21:36+0000 [stdout#info] getaddrinfo error: [Errno -3] Try again
  18.8  19 | c : DNS request from ('10.100.102.2', 48398) to None: 37 bytes
  18.8  19 | c : DNS request from ('10.100.102.2', 59171) to None: 47 bytes
  18.9  12 | 2018-11-07T07:21:36+0000 [-] DNSDatagramProtocol starting on 7848
  18.9  12 | 2018-11-07T07:21:36+0000 [-] Starting protocol <twisted.names.dns.DNSDatagramProtocol object at 0x7f1c6d0704e0>
  18.9  12 | 2018-11-07T07:21:36+0000 [stdout#info] A query: b'beacons4.gvt2.com'
  18.9  12 | 2018-11-07T07:21:36+0000 [stdout#info] A query: b'clients4.google.com'
  18.9  12 | 2018-11-07T07:21:36+0000 [stdout#info] Updated query of type 1 from b'beacons4.gvt2.com.localdomain' to b'beacons4.gvt2.com'
  19.0  12 | 2018-11-07T07:21:36+0000 [stdout#info] A query: b'beacons4.gvt2.com'
  19.0  19 | c : DNS request from ('10.100.102.2', 35063) to None: 49 bytes
  19.0  12 | 2018-11-07T07:21:36+0000 [stdout#info] getaddrinfo error: [Errno -3] Try again
  19.0  12 | 2018-11-07T07:21:36+0000 [-] (UDP Port 29493 Closed)
  19.0  12 | 2018-11-07T07:21:36+0000 [-] Stopping protocol <twisted.names.dns.DNSDatagramProtocol object at 0x7f1c6cf996d8>
  19.0  12 | 2018-11-07T07:21:36+0000 [stdout#info] Updated query of type 1 from b'clients4.google.com.localdomain' to b'clients4.google.com'
  19.0  12 | 2018-11-07T07:21:36+0000 [stdout#info] A query: b'clients4.google.com'
  19.7  19 | c : DNS request from ('10.100.102.2', 53370) to None: 56 bytes
  19.7  12 | 2018-11-07T07:21:37+0000 [stdout#info] A query: b'cuscochromeextension-pa.googleapis.com'
  19.8  19 | c : DNS request from ('10.100.102.2', 39549) to None: 37 bytes
  19.8  12 | 2018-11-07T07:21:37+0000 [stdout#info] A query: b'clients4.google.com'
  21.1 TEL | Main process (bash --norc)
  21.1 TEL |  exited with code 0.
  21.1 TEL | END SPAN __init__.py:530(wait_for_exit)   15.0s
  21.1 TEL | Success. Starting cleanup.
  21.1 >>> | Exit cleanup in progress
  21.1 TEL | (Cleanup) Terminate local process
  21.1 TEL | (Cleanup) Unmount remote filesystem
  21.1 TEL | [32] Running: fusermount -z -u /tmp/tel-k12zhehx/fs
  21.2 TEL | (Cleanup) Kill background items
  21.2 TEL | Killing BackgroundProcess [19] sshuttle
  21.2 TEL | Killing BackgroundProcess [16] SSH port forward (socks and proxy poll)
  21.2 TEL | Killing BackgroundThread Web server for proxy poll
  21.2 TEL | [16] exit 0
  21.2  19 | >> iptables -t nat -D OUTPUT -j sshuttle-12300
  21.2  19 | >> iptables -t nat -D PREROUTING -j sshuttle-12300
  21.2  19 | >> iptables -t nat -F sshuttle-12300
  21.2  19 | >> iptables -t nat -X sshuttle-12300
  21.2  19 |  s: SW#-1:172.30.39.119:443: deleting (13 remain)
  21.2  19 |  s: SW'unknown':Mux#16: deleting (12 remain)
  21.2  19 |  s: SW'unknown':Mux#10: deleting (11 remain)
  21.2  19 |  s: SW#13:172.30.39.119:443: deleting (10 remain)
  21.2  19 |  s: SW#15:172.30.39.119:443: deleting (9 remain)
  21.2  19 |  s: SW'unknown':Mux#11: deleting (8 remain)
  21.2  19 |  s: SW#9:172.30.39.119:443: deleting (7 remain)
  21.2  19 |  s: SW'unknown':Mux#12: deleting (6 remain)
  21.2  19 |  s: SW#14:172.30.39.119:443: deleting (5 remain)
  21.2  19 |  s: SW'unknown':Mux#13: deleting (4 remain)
  21.2  19 |  s: SW'unknown':Mux#14: deleting (3 remain)
  21.2  19 |  s: SW#16:172.30.39.119:443: deleting (2 remain)
  21.2  19 |  s: SW'unknown':Mux#15: deleting (1 remain)
  21.2  19 |  s: SW#17:172.30.39.119:443: deleting (0 remain)
  21.2 TEL | [19] exit -15
  21.2 TEL | Killing BackgroundProcess [13] kubectl port-forward
  21.2 TEL | [13] exit -15
  21.2 TEL | Killing BackgroundProcess [12] kubectl logs
  21.2 TEL | [12] exit -15
  21.2 TEL | Killing BackgroundThread sudo privileges holder
  21.7 TEL | (sudo privileges holder thread exiting)
  21.7 TEL | (Cleanup) Stop time tracking
  21.7 TEL | END SPAN main.py:41(main)   21.7s
  21.7 TEL | SPAN SUMMARY:
  21.7 TEL |   21.7s main.py:41(main)
  21.7 TEL |    0.5s   startup.py:58(__init__)
  21.7 TEL |    0.1s     2 kubectl version --short
  21.7 TEL |    0.0s     3 kubectl config current-context
  21.7 TEL |    0.1s     4 kubectl config view -o json
  21.7 TEL |    0.1s     5 kubectl get ns nemo
  21.7 TEL |    0.0s   6 ssh -V
  21.7 TEL |    0.1s   7 sudo -n echo -n
  21.7 TEL |    0.1s   8 kubectl --context nemo/127-0-0-1:8443/system:admin --namespace nemo get pods t
  21.7 TEL |    0.3s   remote.py:149(get_remote_info)
  21.7 TEL |    0.1s     remote.py:73(get_deployment_json)
  21.7 TEL |    0.1s       9 kubectl --context nemo/127-0-0-1:8443/system:admin --namespace nemo get deploy
  21.7 TEL |    0.1s     10 kubectl --context nemo/127-0-0-1:8443/system:admin --namespace nemo get pod -
  21.7 TEL |    0.1s     remote.py:111(wait_for_pod)
  21.7 TEL |    0.1s       11 kubectl --context nemo/127-0-0-1:8443/system:admin --namespace nemo get pod t
  21.7 TEL |    0.5s   __init__.py:38(connect)
  21.7 TEL |    0.0s     14 ssh -F /dev/null -q -oStrictHostKeyChecking=no -oUserKnownHostsFile=/dev/null
  21.7 TEL |    0.2s     15 ssh -F /dev/null -q -oStrictHostKeyChecking=no -oUserKnownHostsFile=/dev/null
  21.7 TEL |    1.0s   remote_env.py:65(get_remote_env)
  21.7 TEL |    1.0s     17 kubectl --context nemo/127-0-0-1:8443/system:admin --namespace nemo exec tel-
  21.7 TEL |    0.2s   mount.py:32(mount_remote_volumes)
  21.7 TEL |    0.2s     18 sshfs -p 44447 -F /dev/null -o StrictHostKeyChecking=no -o UserKnownHostsFile
  21.7 TEL |    2.3s   vpn.py:235(connect_sshuttle)
  21.7 TEL |    0.0s     vpn.py:79(get_proxy_cidrs)
  21.7 TEL |    2.3s     vpn.py:276(connect_sshuttle,sshuttle-wait)
  21.7 TEL |    0.1s       20 python3 -c 'import socket; socket.gethostbyname('"'"'hellotelepresence0'"'"')
  21.7 TEL |    0.1s       21 python3 -c 'import socket; socket.gethostbyname('"'"'hellotelepresence1'"'"')
  21.7 TEL |    0.1s       22 python3 -c 'import socket; socket.gethostbyname('"'"'hellotelepresence2'"'"')
  21.7 TEL |    0.1s       23 python3 -c 'import socket; socket.gethostbyname('"'"'hellotelepresence3'"'"')
  21.7 TEL |    0.1s       24 python3 -c 'import socket; socket.gethostbyname('"'"'hellotelepresence4'"'"')
  21.7 TEL |    0.1s       25 python3 -c 'import socket; socket.gethostbyname('"'"'hellotelepresence5'"'"')
  21.7 TEL |    0.1s       26 python3 -c 'import socket; socket.gethostbyname('"'"'hellotelepresence6'"'"')
  21.7 TEL |    0.1s       27 python3 -c 'import socket; socket.gethostbyname('"'"'hellotelepresence7'"'"')
  21.7 TEL |    0.1s       28 python3 -c 'import socket; socket.gethostbyname('"'"'hellotelepresence8'"'"')
  21.7 TEL |    0.1s       29 python3 -c 'import socket; socket.gethostbyname('"'"'hellotelepresence9'"'"')
  21.7 TEL |    0.1s       30 python3 -c 'import socket; socket.gethostbyname('"'"'hellotelepresence10'"'"'
  21.7 TEL |    0.1s       31 python3 -c 'import socket; socket.gethostbyname('"'"'hellotelepresence11'"'"'
  21.7 TEL |   15.0s   __init__.py:530(wait_for_exit)
  21.7 TEL |    0.0s   32 fusermount -z -u /tmp/tel-k12zhehx/fs
  21.7 TEL | (Cleanup) Remove temporary directory
  21.7 TEL | (Cleanup) Save caches
